cypress/e2e/web/createTechnology.js

```diff
@@ -57,6 +57,7 @@ describe('technology input form help', () => {
 describe('creating/editing technology', () => {
 	beforeEach(() => {
 		cy.authenticate().visit('/technology/new');
+		cy.intercept('GET', '/user/me', { fixture: 'user' });
```

Comment:
we cant forget to get rid of this 😅

Reply:
get rid of what? the fixtures or the requests being made indeed?

Reply:
@marcusmota doesn't sounds good for me the fact that we're using a kind of "mock" in a E2E, this can leads to future hide of bugs regarding user object